src/silo/common/nucleotide_symbols.test.cpp

```diff
@@ -0,0 +1,25 @@
+#include "silo/common/nucleotide_symbols.h"
+
+#include <gtest/gtest.h>
+
+TEST(genome_symbol, enum_should_have_same_length_as_symbol_representation) {
+   EXPECT_EQ(silo::SYMBOL_COUNT, silo::SYMBOL_REPRESENTATION.size());
```

Comment:
I had a look. At first, i thought we could delete it. However it is used in sequence_store.h to generate the array. I think it is best to leave it in for better readability, since we would otherwise need to write: `static_cast<unsigned>(NUCLEOTIDE_SYMBOL::N) + 1`.